### ISSUE TITLE
extensions/Training_PRO contains two differently cased README files

### ISSUE THREAD

**Issue by orlandov** [oobabooga/text-generation-webui#4075]
### Describe the bug

Recently a new file was introduced into `extensions/Training_PRO` duplicates the filename of an existing file with different case (`readme.md`). At least on my system, this causes git in WSL1 to freak out, and the files become very difficult to clean up (even for an experienced git user), and many operations in git will not longer work when the repo enters this state. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

```
❯ git config --global core.ignorecase
false
❯ git clone https://github.com/oobabooga/text-generation-webui tgwui1
Cloning into 'tgwui1'...
remote: Enumerating objects: 12546, done.
remote: Counting objects: 100% (2671/2671), done.
remote: Compressing objects: 100% (612/612), done.
remote: Total 12546 (delta 2234), reused 2379 (delta 2052), pack-reused 9875
Receiving objects: 100% (12546/12546), 11.07 MiB | 16.67 MiB/s, done.
Resolving deltas: 100% (8541/8541), done.
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'extensions/Training_PRO/README.md'
  'extensions/Training_PRO/readme.md'
❯ git config --global core.ignorecase true
❯ git config --global core.ignorecase
true
❯ git clone https://github.com/oobabooga/text-generation-webui tgwui2
Cloning into 'tgwui2'...
remote: Enumerating objects: 12546, done.
remote: Counting objects: 100% (2774/2774), done.
remote: Compressing objects: 100% (589/589), done.
remote: Total 12546 (delta 2340), reused 2507 (delta 2178), pack-reused 9772
Receiving objects: 100% (12546/12546), 11.00 MiB | 16.86 MiB/s, done.
Resolving deltas: 100% (8545/8545), done.
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'extensions/Training_PRO/README.md'
  'extensions/Training_PRO/readme.md'
```

### Logs

```shell
provided
```

### System Info

```shell
WSL1 on Windows 10
```

**Comment by jamesbraza** [comment]
Also hitting this on macOS, it's not just a Windows thing.  It's annoying, I don't even know how this happened

**Comment by orlandov** [comment]
Fixed via #4112

**Comment by FartyPants** [comment]
Damn. Stinky windoze
